<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenGL Test 1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="GLWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Histogram" name="widget_2" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>150</height>
       </size>
      </property>
      <widget class="QLabel" name="isoLabel">
       <property name="geometry">
        <rect>
         <x>88</x>
         <y>0</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="minLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="maxLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>500</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <widget class="QPushButton" name="browse_pushButton">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>34</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Point_size_comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>410</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="Camera_label">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>260</y>
         <width>48</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Colour_map_comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>318</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>blue-cyan-green-magenta-red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue-red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>grey scale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>red-yellow-green-cyan-blue-magenta-red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green-yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green-magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue-green-red-green-blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white-black-white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>red-blue-cyan-magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue-cyan-grren-yellow-white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>landscape</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>constant</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>69</y>
         <width>158</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Grid</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="Colourmap">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>318</y>
         <width>70</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Colour map</string>
       </property>
      </widget>
      <widget class="QSlider" name="opacitySlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="Point_size_label">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>410</y>
         <width>59</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Point Size</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Drawing_mode_comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>289</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Points Above</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Points Below</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lines Above</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lines Below</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Isopoints</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Isosurface</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>160</y>
         <width>105</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Bounding Box</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>100</y>
         <width>94</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto rotate</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>158</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto ramp isolevel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="filepath_Edit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>34</y>
         <width>158</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QComboBox" name="Camera_combo">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Front</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axonometric</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>69</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Axes</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="Grid_resolution_comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>381</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="Grid_resolution_label">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>381</y>
         <width>89</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Grid resolution</string>
       </property>
      </widget>
      <widget class="QLabel" name="DrawingMode">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>289</y>
         <width>87</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Drawing Mode</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>59</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Background_comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>350</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="HorizontalSlider">
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="sliderPosition">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Histogram</class>
   <extends>QWidget</extends>
   <header>histogram.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
